craft_parts/ctl.py

```diff
@@ -70,8 +94,33 @@ def client(cmd: str, args: List[str]):
         fifo.write(json.dumps(data))
 
     with open(feedback_fifo, "r") as fifo:
-        feedback = fifo.readline().strip()
+        feedback = fifo.readline().split(" ", 1)
+
+    status = feedback[0]
+    message = feedback[1].strip() if len(feedback) > 1 else ""
+    retval = None
+
+    if status == "OK":
```

Comment:
can you add a comment about what sends the OK and ERR (I can tell in this PR, but in the future I might certainly  forget where this comes from :-))

Reply:
Added a reminder note.